neso_solar_consumer/app.py

```diff
@@ -0,0 +1,101 @@
+"""
```

Comment:
I can see this file on the main branch already? does this branch just need updating?

Reply:
Yeah I think I needed updating, I think now it's fine.